MOLT 1.2.4 release notes (#19504)
cockroachdb/docs: src/current/_includes/molt/replicator-flags.md

| `--scanSize`                           | `INT`      | The number of rows to retrieve from the staging database used to store metadata for [replication modes](#fetch-mode).<br><br>**Default:** `10000`                                                                                                                                                                                                     |
| `--schemaRefresh`                      | `DURATION` | How often a watcher will refresh its schema. If this value is zero or negative, refresh behavior will be disabled.<br><br>**Default:** `1m0s`                                                                                                                                                                                                         |
| `--sourceConn`                         | `STRING`   | The source database's connection string.                                                                                                                                                                                                                                                                                                              |

> | `--stageDisableCreateTableReaderIndex` | `BOOL`     | Disable the creation of partial covering indexes to improve reads on staging tables. Set to `true` if creating indexes on existing tables will have a large operational impact.<br><br>**Default:** `false`                                                                                                                                          |

Reviewer: non-blocking suggestion, courtesy of Reviewer GPT:
```suggestion
| `--stageDisableCreateTableReaderIndex` | `BOOL`     | Disable the creation of partial covering indexes to improve read performance on staging tables. Set to `true` if creating indexes on existing tables would cause a significant operational impact.<br><br>**Default:** `false`                                                                                                                                          |
```

> | `--stageDisableCreateTableReaderIndex` | `BOOL`     | Disable the creation of partial covering indexes to improve reads on staging tables. Set to `true` if creating indexes on existing tables will have a large operational impact.<br><br>**Defaulty:** `false`                                                                                                                                          |

Reviewer 2: Nit: typo, should be `Default` instead of `Defaulty`
